src/models/room_member.rs

```diff
@@ -58,11 +58,26 @@ pub struct RoomMember {
     /// The human readable name of this room member.
     pub name: String,
     /// The events that created the state of this room member.
+    #[serde(skip)]
```

Comment:
If we're going to skip this we should explain why in a comment here. Perhaps explain the implications of this in the state store docs.

Reply:
I skipped  the event vectors just to get everything working (they don't impl deserialize). There is a serde trick to impl ser/de for types outside your crate would this be preferable.